dev/dll/DensityStyles/Compact.xaml

```diff
@@ -1,14 +1,14 @@
-<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
 <ResourceDictionary 
   xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" 
-  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">   
+  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
     <Style TargetType="TextBlock" >
         <Setter Property="FontSize" Value="14" />
     </Style>
     <x:Double x:Key="ControlContentThemeFontSize">14</x:Double>
     <x:Double x:Key="ContentControlFontSize">14</x:Double>
     <x:Double x:Key="TextControlThemeMinHeight">24</x:Double>
-    <Thickness x:Key="TextControlThemePadding">2,1,6,0</Thickness>
+    <Thickness x:Key="TextControlThemePadding">2,2,6,1</Thickness>
```

Comment:
I think this is only for compact, we also need to update the value for non-compact case.

You probably need to add the resource in Common_themeresources like you did for TextControlBorderThemeThickness and TextControlBorderThemeThicknessFocused.

Reply:
I added and adjusted the `TextControlThemePadding` here e75f724. Can you take a look at it and make sure the height and positioning is good?

Reply:
The default value is `10,3,6,6` so I added one pixel at the top and bottom

Reply:
Looks like it's not centered. Adding one pixel at top and bottom feels right to me, but the visual doesn't look right for some reason. Maybe because we are changing BorderThickness dynamically in StoryBoard? Can you double check why the position is off?

![20190812123812](https://user-images.githubusercontent.com/4424330/62893084-917b4200-bcfe-11e9-81cd-4ebf743b6b8a.png)

Reply:
@kaiguo I actually can't get a repro of that 
![image](https://user-images.githubusercontent.com/11960777/62900729-c774f200-bd0f-11e9-9b48-bdbc3d8fd66d.png)
Which page is that?

Reply:
Comparison btw the old:
![image](https://user-images.githubusercontent.com/11960777/62901219-140cfd00-bd11-11e9-898c-1fc8fdad945d.png)
And the new:
![image](https://user-images.githubusercontent.com/11960777/62901359-677f4b00-bd11-11e9-812a-6970fb8e88df.png)

Reply:
Must be a bad build or something. I did a clean build and it looks good to me now. Sorry about the confusion.